Formula/grafana.rb

```diff
@@ -0,0 +1,154 @@
+require "language/node"
+
+class Grafana < Formula
+  desc "Gorgeous metric visualizations and dashboards for timeseries databases."
+  homepage "http://grafana.org"
+  url "https://github.com/grafana/grafana/archive/v3.0.4.tar.gz"
+  sha256 "f26a374326e64a8f83c57fdf916ba1c8524dd55002dfe628b4854d05fed3715a"
+
+  head "https://github.com/grafana/grafana.git"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    grafana_path = buildpath/"src/github.com/grafana/grafana"
+    grafana_path.install Dir["*"]
+    grafana_path.install ".jscs.json", ".jsfmtrc", ".jshintrc", ".bowerrc"
+
+    cd grafana_path do
+      # The sass-lint npm package dependencey in package.json specifies any
+      # version greater than 1.6.0 for grafana, but on OS X versions 1.8.0+
+      # break. This replaces the specification for any version >= 1.6.0 and
+      # sets it to 1.7.0 (which works).
+      inreplace "package.json", '"sass-lint": "^1.6.0"', '"sass-lint": "1.7.0"'
```

Comment:
That's actually happened with the next version, so this isn't necessary anymore. I wasn't sure about changing the version on this in the middle of getting it accepted, but I'll updated all that accordingly (and take this chunk out entirely now that it's not relevant).